src/workers/walletWorker.ts

```diff
@@ -102,52 +102,63 @@ export async function setupMutinyWallet(
     nsec?: string
 ): Promise<boolean> {
     // initialize both inside worker and outside
-    if (reportDiagnostics && sentryenv !== "") {
-        Sentry.init({
-            dsn: "https://192c556849619517322719962a057376@sen.mutinywallet.com/2",
-            environment: sentryenv,
-            release: "mutiny-web@" + RELEASE_VERSION,
-            integrations: [
-                Sentry.captureConsoleIntegration(), // grab all mutiny-node console lines
-                Sentry.browserTracingIntegration(),
-                Sentry.replayIntegration()
-            ],
-
-            initialScope: {
-                tags: { component: "worker" }
-            },
-
-            // ignore any hex larger than 20 char
-            ignoreErrors: [/(?:[0-9a-fA-F]{20,}\b)+/],
-
-            // only do a new issue for errors w/ or w/o exceptions, and warnings
-            beforeSend(event, hint) {
-                const error = hint.originalException;
-                if (error && typeof error === "object" && "message" in error) {
-                    return event;
-                } else if (event.level == "warning" || event.level == "error") {
-                    return event;
-                } else {
-                    return null;
-                }
-            },
-
-            // Set tracesSampleRate to 1.0 to capture 100%
-            // of transactions for performance monitoring.
-            // We recommend adjusting this value in production
-            tracesSampleRate: 1.0,
-
-            // Set `tracePropagationTargets` to control for which URLs distributed tracing should be enabled
-            tracePropagationTargets: [
-                "localhost",
-                /^https:\/\/mutinywallet\.com/
-            ],
-
-            // Capture Replay for 10% of all sessions,
-            // plus 100% of sessions with an error
-            replaysSessionSampleRate: 0.1,
-            replaysOnErrorSampleRate: 1.0
-        });
+    try {
+        if (reportDiagnostics && sentryenv !== "") {
```

Comment:
Password is here too, do you not want to also do the same logic you do with the megastore?